flink-contrib/src/main/java/org/apache/flink/contrib/tweetinputformat/io/SimpleTweetInputFormat.java

```diff
@@ -0,0 +1,67 @@
+package org.apache.flink.contrib.tweetinputformat.io;
+
+import org.apache.flink.api.common.io.DelimitedInputFormat;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.java.typeutils.GenericTypeInfo;
+import org.apache.flink.api.java.typeutils.ResultTypeQueryable;
+import org.apache.flink.contrib.tweetinputformat.model.tweet.Tweet;
+import org.codehaus.jackson.JsonParseException;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+
+public class SimpleTweetInputFormat extends DelimitedInputFormat<Tweet> implements ResultTypeQueryable<Tweet> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SimpleTweetInputFormat.class);
+
+    @Override
+    public Tweet nextRecord(Tweet record) throws IOException {
+        Boolean result = false;
+
+        do {
+            try {
+                record.reset(0);
+                record = super.nextRecord(record);
+                result = true;
+
+            } catch (JsonParseException e) {
+                result = false;
+
+            }
+        } while (!result);
+
+        return record;
+    }
+
+    @Override
+    public Tweet readRecord(Tweet reuse, byte[] bytes, int offset, int numBytes) throws IOException {
+
+
+        InputStreamReader jsonReader = new InputStreamReader(new ByteArrayInputStream(bytes));
+        jsonReader.skip(offset);
+
+        JSONParser parser = new JSONParser();
```

Comment:
I know, I have tried in the beginning to create them as instance fields in the class, however I received an error because the fields are not serializable. I tried to declare them as transient, but Flink threw "Can not submit Job" exception, If you have suggestions please let me know